pandas/tests/indexes/datetimes/test_scalar_compat.py

```diff
@@ -64,6 +64,39 @@ def test_dti_timestamp_fields(self, field):
             result = getattr(Timestamp(idx[-1]), field)
         assert result == expected
 
+    @pytest.mark.parametrize(
+        "tz,expected_vals",
+        [
+            (
+                "utc",
+                [
+                    "2000-01-01T00:00:00.000000000Z",
+                    "2000-01-02T00:00:00.000000000Z",
+                    "2000-01-03T00:00:00.000000000Z",
+                ],
+            ),
+            # "US/Eastern",
```

Comment:
pls mangle to ensure this fails linting.  want to avoid accidentally merging commented-out code

Reply:
Yea my hope is to get this working; just a stub for now

Reply:
FWIW I think should be possible to support timezones if we vendor updates from Numpy:

https://github.com/numpy/numpy/blob/ea66b1d154a99ed7e4325150ef1bf509b7a2b268/numpy/core/src/multiarray/datetime_strings.c#L873

Reply:
For a while I've been wanting to make a PR at numpy to get them to expose these functions so we could `#include` them and not re-implement them.  Every time I've trird, my C-fu has not been up to the task.  You might have better luck, if you're interested.

Reply:
Cool I’ll check out their repo. Would be nice instead of having to vendor